community-spotlight-simplifying-license-management-aws-xata.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Simplifying license management and reducing costs with AWS and Xata'
+description: 'Putting the spotlight on Ilia Gandelman, who significantly improved a cumbersome software license management system for smart rehabilitation therapy devices using Xata and AWS.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/community-spotlight-simplifying-license-management.jpg
+  alt: Community Spotlight
+author: Cezzaine Zaher
+date: 09-05-2024
```

Comment:
```suggestion
date: 05-09-2024
```

@CezzaineZ I just noticed the date that we'll need to fix for sorting purposes. These dates are formatted in the US format actually. So it reads September 5th at the moment. I think we need to swap these.